src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.unit.netcoreapp.cs

```diff
@@ -0,0 +1,520 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.ObjectModel;
+using System.Threading;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class FileSystemWatcherFilterListTests : FileSystemWatcherTest
+    {
+        [Fact]
+        public void DefaultFiltersValue()
+        {
+            var watcher = new FileSystemWatcher();
+            Assert.Equal(0, watcher.Filters.Count);
+            Assert.Empty(watcher.Filters);
+            Assert.NotNull(watcher.Filters);
+            EqualItems(new Collection<string> { }, watcher.Filters);
```

Comment:
These should all be `Assert.Equal` with the expected being `new string [] {}`. That should get the right overload, if it doesn't, cast to `IEnumerable<string>()`- but only if you have to.